test/integration/targets/ansible-galaxy/aliases

```diff
@@ -1,4 +1,5 @@
 destructive
 shippable/posix/group3
 skip/python2.6  # build uses tarfile with features not available until 2.7
+skip/python2.7  # requires python 3 for twisted dependencies
```

Comment:
Tests need to continue supporting Python 2.7, so avoid dependencies that are Python 3+ only.

```suggestion
```

Perhaps you can use `SimpleHTTPServer` as we do in other tests?

Reply:
I think I tried that but couldn't get it to work effectively for a reason that escapes my mind now. Will try and pick this up again and re-implement with that, though.

Reply:
Believe I've resolved this in the latest revision (pending CI)